app/src/main/java/com/example/easyupipayment/MainActivity.java

```diff
@@ -29,16 +32,22 @@ protected void onCreate(Bundle savedInstanceState) {
         imageView = findViewById(R.id.imageView);
         statusView = findViewById(R.id.textView_status);
         payButton = findViewById(R.id.button_pay);
-
+        Vpa=findViewById(R.id.edittext_vpa);
+        Pname=findViewById(R.id.edittext_pname);
+        Tid=findViewById(R.id.edittext_tid);
+        Trefid=findViewById(R.id.edittext_trefid);
+        Desc=findViewById(R.id.edittext_desc);
```

Comment:
Coding style should be neat. Please use camel case for variables and keep spaces in assignment.

Reply:
I will keep this suggestion in mind while doing next PR

Reply:
You don't have any neee to create seperate PR. Just push your local commits to `master` branch of your repo. All changes/commits will be automatically included in this PR.